api/v1/controllers/HealthController.js

```diff
@@ -0,0 +1,19 @@
+var bodyParser = require('body-parser');
+var middleware = require('../middleware');
+
+var router = require('express').Router();
+
+function healthCheck(req, res, next) {
+    next();
+    return null;
+}
+
+router.use(bodyParser.json());
+router.use(middleware.auth);
```

Comment:
You won't need to use the `auth` middleware here; there's nothing in this controller that makes use of any authentication/authorization information.